### ISSUE TITLE
Java SDK does not support G6E instance types

### ISSUE THREAD

**Issue by yaronr** [aws/aws-sdk-java-v2#5606]
### Describe the bug

The code below doesn't return G6E instance type data for relevant regions (us-east-1,2 west-2):
```
    DescribeInstanceTypesRequest describeInstanceTypesRequest = DescribeInstanceTypesRequest.builder()
                                                                                            .build();
    try {
      DescribeInstanceTypesIterable responseIter = ec2.describeInstanceTypesPaginator(describeInstanceTypesRequest);
```

### Regression Issue

- [ ] Select this option if this issue appears to be a regression.

### Expected Behavior

The SDK should report G6E instance types for relevant regions

### Current Behavior

The SDK does not report G6E instance types for relevant regions

### Reproduction Steps

Run the above code, or take a look at:
https://raw.githubusercontent.com/aws/aws-sdk-java-v2/master/services/ec2/src/main/resources/codegen-resources/service-2.json

### Possible Solution

Thre's PR #5553

### Additional Information/Context

Appreciate if you could expedite this, it's holding our clients back from using G6E instance typesx.

### AWS Java SDK version used

2.28.2

### JDK version used

21

### Operating System and version

Amazon Linux 2023

**Comment by bhoradc** [comment]
Hello @yaronr, 

Thanks again for bringing this issue to our attention and for your understanding regarding the processes involved in updating the AWS SDKs.

We have reached out to the EC2 service team responsible for maintaining the service model, and they are aware of the need to add the `G6E` instance types to the InstanceType enum in the SDKs. 

I cannot provide a timeline for when these changes will be available in the SDK release. But can assure that the team is actively working on incorporating these changes.

Thank you for the patience and understanding. 

Regards, 
Chaitanya

**Comment by bhoradc** [comment]
Hi @yaronr, 

`G6E` EC2 instance types are now available in Java SDK starting version 2.28.7, refer [here](https://raw.githubusercontent.com/aws/aws-sdk-java-v2/master/services/ec2/src/main/resources/codegen-resources/service-2.json). Therefore, I will go ahead and close this issue. Thanks again for reporting it. 

Regards,
Chaitanya
